### PR TITLE
[PLATFORM BUSINESS APPS] -  SEER ODS Regression updates 

### PR DESCRIPTION
**Pull request by Mariachaudhry**

### CODE REVIEW

**Review by Mariachaudhry — COMMENTED** [review]
Reviewed by Maria

**Review by andreyeve — APPROVED** [review]

File: src/test/java/ServiceNow/PlatformBusinessApps/NERD/Features/NCI Staff Members.feature [hunk 1]
```diff
@@ -30,10 +30,18 @@ Feature: NCI Staff Member Test Scenarios
...
     When NCI Staff Member submits new COVID19 Activities submission
-#    And DOC Planning Contact returning the COVID-19 Activities submission back to NCI Staff Member
-#    And NCI Staff Member views the returned COVID-19 Activities submission
-#    Then NCI Staff Member DOES NOT see the 'Delete' button for COVID-19 Activities submission returned record
\ No newline at end of file
+    And DOC Planning Contact returning the COVID19 Activities submission back to NCI Staff Member
+    And NCI Staff Member views the returned COVID19 Activities submission
+    Then NCI Staff Member DOES NOT see the 'Delete' button for COVID19 Activities submission returned record
+
+  @NERDSNOW-690 @chaudhryma  @selenium @In-Progress
```
Comment (matakevin) [inline src/test/java/ServiceNow/PlatformBusinessApps/NERD/Features/NCI Staff Members.feature:41]: delete extra space between your name tag and the selenium tag

File: src/test/java/ServiceNow/PlatformBusinessApps/NERD/Features/NCI Staff Members.feature [hunk 2]
```diff
@@ -30,10 +30,18 @@ Feature: NCI Staff Member Test Scenarios
     And NCI Staff Member views the returned Collaborations submission
     Then NCI Staff Member DOES NOT see the 'Delete' button for Collaborations submission returned record
 
-  @NERDSNOW-689 @chaudhryma  @In-Progress @selenium
+  @NERDSNOW-689 @chaudhryma  @Regression @selenium
```
Comment (matakevin) [inline src/test/java/ServiceNow/PlatformBusinessApps/NERD/Features/NCI Staff Members.feature:33]: delete extra space between your name tag and the Regression tag

**Review by matakevin — CHANGES_REQUESTED** [review]
Left some comments on a feature file for extra spaces but aside from that it looks good.

**Review by matakevin — APPROVED** [review]
LGTM

File: src/test/java/ServiceNow/PlatformBusinessApps/SEER/SEER_Runners/Run_SEER_Progression_Test.java [hunk 3]
```diff
@@ -3,11 +3,11 @@
 import io.cucumber.testng.AbstractTestNGCucumberTests;
 import io.cucumber.testng.CucumberOptions;
 
-@CucumberOptions(plugin = {"html:target/html-reports/cucumber-default-report", "json:target/cucumber.json",
-        "junit:target/cucumber.xml", "rerun:target/failed.txt",
+@CucumberOptions(plugin = {"html:target/html-reports/cucumber-default-report",
```
Comment (hamidnesar) [inline src/test/java/ServiceNow/PlatformBusinessApps/SEER/SEER_Runners/Run_SEER_Progression_Test.java:6]: Please add .html for the html report. i.e: _"html:target/html-reports/cucumber-default-report.html"_

File: src/test/java/ServiceNow/PlatformBusinessApps/SEER/SEER_Runners/Run_SEER_Regression_Test.java [hunk 4]
```diff
@@ -3,11 +3,11 @@
 import io.cucumber.testng.CucumberOptions;
 import io.cucumber.testng.AbstractTestNGCucumberTests;
 
-@CucumberOptions(plugin = {"html:target/html-reports/cucumber-default-report", "json:target/cucumber.json",
-        "junit:target/cucumber.xml", "rerun:target/failed.txt",
+@CucumberOptions(plugin = {"html:target/html-reports/cucumber-default-report",
```
Comment (hamidnesar) [inline src/test/java/ServiceNow/PlatformBusinessApps/SEER/SEER_Runners/Run_SEER_Regression_Test.java:6]: Please add .html for the html report. i.e: "html:target/html-reports/cucumber-default-report.html"

**Review by hamidnesar — CHANGES_REQUESTED** [review]
Hi, please review the comments.

**Review by hamidnesar — APPROVED** [review]
Reviewed and approved.

**Pull request merged**